compose/ui/ui/src/jsWasmMain/kotlin/androidx/compose/ui/draganddrop/DragAndDrop.jsWasm.kt

```diff
@@ -21,14 +21,14 @@ import androidx.compose.ui.geometry.Offset
 /**
  * A representation of an event sent by the platform during a drag and drop operation.
  */
-actual class DragAndDropEvent
+actual class DragAndDropEvent(val offset: Offset)
```

Comment:
Please mark the constructor `internal`. Users are not supposed to create instances of this class